# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1548229593 28800
#      Tue Jan 22 23:46:33 2019 -0800
# Node ID 884fdd6c92104aea33c3193d2449974736cf4ca6
# Parent  5224600b1b201337c31b576e05341e58328633f7
fix declaration of warn_divide_by_zero deprecation wrapper (bug #46650)

* errwarn.h (warn_divide_by_zero): Make function declaration inline and
conform to Octave coding style.

diff --git a/libinterp/corefcn/errwarn.h b/libinterp/corefcn/errwarn.h
--- a/libinterp/corefcn/errwarn.h
+++ b/libinterp/corefcn/errwarn.h
@@ -179,14 +179,16 @@ warn_invalid_value_specified (const char
 OCTINTERP_API extern void
 warn_logical_conversion (void);
 
 OCTINTERP_API extern void
 warn_wrong_type_arg (const char *name, const octave_value& tc);
 
 #if defined (OCTAVE_USE_DEPRECATED_FUNCTIONS)
 
-OCTAVE_DEPRECATED (6, "do not use 'Octave:divide-by-zero'")
-OCTINTERP_API extern void warn_divide_by_zero (void) {}
+OCTAVE_DEPRECATED (6, "this function will be removed in a future version of Octave")
+inline void
+warn_divide_by_zero (void)
+{ }
 
 #endif
 
 #endif
